model/task_lifecycle.go

```diff
@@ -523,6 +524,15 @@ func UpdateBlockedDependencies(t *task.Task) error {
 		return errors.Wrapf(err, "can't get tasks depending on task '%s'", t.Id)
 	}
 
+	if t.IsPartOfSingleHostTaskGroup() && len(dependentTasks) > 0 {
+		grip.Info(message.Fields{
+			"message": "blocked task group dependent tasks",
+			"ticket":  "EVG-12923",
+			"task":    t.Id,
+			"stack":   debug.Stack(),
```

Comment:
I don't think there's any functional difference, but you can also log a message.NewStack which has a couple of configuration options if you need

Reply:
done